types/p2p.yaml

```diff
@@ -58,6 +58,7 @@
 
   ENR:
     type: string
+    nullable: true
```

Comment:
Can you add this to
Peer.properties.peer_id.enr since this enr type might be used in some other api endpoints where it isn't necessary nullable.

Syntax is:
```
enr:
   allOf:
      - $ref: "./p2p.yaml#/ENR"
      - nullable: true
```

Reply:
done. looked at the local server and it seems to mark both endpoints nullable, so looks ok.